### ISSUE TITLE
Feature - Improved accessibility

### ISSUE THREAD

**Issue by mellybess** [mellybess/oss-complete#16]
Although this product is good in general, some users may have a hard time due to lack of accessibility:
Contrast between colors should be higher for better accessibility. When text and images are large and/or enlargeable, it is easier for users with poor sight to read and understand the content. When links are underlined (or otherwise differentiated) as well as colored, this ensures that color blind users will be able to notice them. When clickable links and areas are large, this helps users who cannot control a mouse with precision. When pages are not coded in a way that hinders navigation by means of the keyboard alone, or a single switch access device alone, this helps users who cannot use a mouse or even a standard keyboard.

Thanks!
